ebs_cleanup/app.py

```diff
@@ -143,7 +143,7 @@ def lambda_handler(event, context):
     try:
         unattached = []
         min_age = os.environ.get('ebsMinimumAge', '5m')
-        client = boto3.client('ec2')
+        client = boto3.client('ec2', region_name='us-east-1')
```

Comment:
region probably shouldn't be hard coded.  You can just retrieve the region from lambda runtime env.  https://www.radishlogic.com/aws/lambda/aws-lambda-getting-the-runtime-region-via-python/

Reply:
oh, thanks, that's much better

Reply:
I started to implement using `AWS_REGION`, but ran into another local test failure and realized I should be mocking the needed env var, so I backed out the changes to the app and added `AWS_DEFAULT_REGION` to the test.